src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
```

Comment:
You are using a raw type for the MachineProducer interface. It's a good practice to always parametrize interfaces and classes if it is possible. Please, specify the type of machines this producer is producing, like this: `public class BulldozerProducer implements MachineProducer<Bulldozer>`.